build.gradle

```diff
@@ -54,10 +54,10 @@ nexusStaging {
 subprojects {
     repositories {
         maven {
-            url  "https://cardinalcommerce.bintray.com/android"
+            url "https://cardinalcommerceprod.jfrog.io/artifactory/android"
             credentials {
-                username 'braintree-team-sdk@cardinalcommerce'
-                password '220cc9476025679c4e5c843666c27d97cfb0f951'
+                username 'braintree_team_sdk'
```

Comment:
Should there be a CHANGELOG entry for the credentials change? It's more of an integration change on the merchant side so not sure

Reply:
Hmm... I think it would be helpful to call it out in the changelog. It could be a note underneath the changelog entries for the next version that calls out that merchants using 3DS will need to update their credentials.